src/clj/rems/application/master_workflow.clj

```diff
@@ -55,6 +56,7 @@
 
 (def ^:private handler-returned-commands
   (disj handler-all-commands
+        :application.command/promote-to-applicant ;; TODO is this right?
```

Comment:
changing the applicant of a returned application seems a bit weird, but perhaps the application has been returned to e.g. a user who can't log in any more?

Reply:
I felt like changing the applicant of a returned application would have more corner cases to test, but perhaps it's not so.

Reply:
At least, there is no need for a `past-applicants`?

Reply:
Yeah, I don't think there is a need for past-applicants currently, and it's easy to add to the projection later if needed.